app/controllers/ConfirmationController.scala

```diff
@@ -55,4 +58,12 @@ class ConfirmationController @Inject()(
           Redirect(routes.JourneyRecoveryController.onPageLoad())
       }
     }
+
+  //TODO: could be made simpler when consignee flow built?
+  private def hasConsigneeChanged()(implicit request: DataRequest[_]): Boolean =
+    request.userAnswers.get(ConsigneeExcisePage) != ConsigneeExcisePage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeExemptOrganisationPage) != ConsigneeExemptOrganisationPage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeBusinessNamePage) != ConsigneeBusinessNamePage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeAddressPage) != ConsigneeAddressPage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeExportVatPage) != ConsigneeExportVatPage.getValueFromIE801
```

Comment:
Move to `ConsigneeSection`

Reply:
Done in [19d004f](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/23/commits/19d004f1f89c3d1be1adc6408ecf135d81e5e31d)